islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -147,6 +148,15 @@
       <xsl:with-param name="pid" select="$pid"/>
       <xsl:with-param name="datastream" select="$datastream"/>
     </xsl:call-template>
+
+    <!-- Potential qualified date range for dateCreated. -->
+    <xsl:if test="mods:dateCreated">
+      <xsl:call-template name="qualified_date_range">
+        <xsl:with-param name="prefix" select="$prefix"/>
+        <xsl:with-param name="suffix" select="$suffix"/>
+        <xsl:with-param name="value" select="normalize-space(mods:dateCreated/text())"/>
+      </xsl:call-template>
+    </xsl:if>
```

Comment:
Not sure if it could be an issue, but I'm not sure this would behave appropriately if a document had more than one `dateCreated` field in a single `originInfo`... Would end up trying to parse the glob of each? Could either `xsl:for-each` (boo) or `xsl:apply-templates` (yeah!) with a custom `mode`, to handle each hypothetical `dateCreated` element... So something like:
```xml
<xsl:apply-templates select="mods:dateCreated" mode="qualified_mods_date_range">
  [... the prefix and suffix with-param bits ...]
</xsl:apply-templates>
```
and up above:
```xml
<xsl:template match="mods:dateCreated" mode="qualified_mods_date_range" name="qualified_date_range">
    <xsl:param name="prefix"/>
    <xsl:param name="suffix"/>
    <!-- default to the text of the node currently being processed... So if _passed_, will get used... or if not, will get calculated. -->
    <xsl:param name="value" select="normalize-space(.)">
    [...]
</xsl:template>
```